src/train_ussd_public.js

```diff
@@ -17,101 +17,71 @@ go.app = function() {
+                case "state_end_optout":
+                    return $("Thank you. You will no longer receive messages");
+                case "state_end_loss":
+                    return $("We are sorry for your loss. You will no longer receive messages. Should you need support during this difficult time, please contact your local CHEW.");
+                case "state_end_exit":
+                    return $("Thank you for using the Hello Mama service");
+            }
         };
 
-        var errors = {
-            "state_registered_msisdn":
-                $("Mobile number not registered."),
-            "state_msisdn_permission":
-                $("Sorry not a valid input. Welcome to Hello Mama. Do you have permission to manage the number {{msisdn}}?"),
-            "state_language":
-                $("Sorry not a valid input. Welcome to Hello Mama. Please choose your language"),
-            "state_main_menu":
-                $("Sorry not a valid input. Select:"),
-            "state_main_menu_household":
-                $("Sorry not a valid input. Select:"),
-            "state_already_registered_baby":
-                $("Sorry not a valid input. You are already registered for baby messages."),
-            "state_change_menu_sms":
-                $("Sorry not a valid input. Please select what you would like to do:"),
-            "state_voice_days":
-                $("Sorry not a valid input. We will call twice a week. On what days would the person like to receive messages?"),
-            "state_voice_times":
-                $("Sorry not a valid input. Thank you. At what time would they like to receive these calls?"),
-            "state_change_menu_voice":
-                $("Sorry not a valid input. Please select what you would like to do:"),
-            "state_new_msisdn":
-                $("Sorry not a valid input. Please enter the new mobile number you would like to receive weekly messages on. For example, 0803304899"),
-            "state_number_in_use":
-                $("Sorry not a valid input. Sorry, this number is already registered. They must opt-out before they can register again."),
-            "state_msg_language":
-                $("Sorry not a valid input. What language would this person like to receive these messages in?"),
-            "state_optout_reason":
-                $("Sorry not a valid input. Please tell us why you no longer want to receive messages so we can help you further."),
-            "state_loss_subscription":
-                $("Sorry not a valid input. We are sorry for your loss. Would you like to receive a small set of free messages from Hello Mama that could help you in this difficult time?"),
-            "state_optout_receiver":
-                $("Sorry not a valid input. Who would you like to stop receiving messages?"),
+        var state_error_types = {
+            "invalid_selection": "Sorry, invalid option. ",
```

Comment:
Making changes to this returns the good old `object Object` issue 👎  - leave it with me